Dockerfile

```diff
@@ -15,14 +22,12 @@ COPY main.go main.go
 COPY apis/ apis/
 COPY go.mod .
 
-RUN export GOOS=$(echo ${TARGETPLATFORM} | cut -d / -f1) && \
```

Comment:
Changing the variables used to assign these values is a backwards incompatible change. Will we be breaking anyone by making this change?

Reply:
Aren't these set to same values as before, it's just without parsing `TARGETPLATFORM`. Why do you think this is backwards incompatible?

Reply:
It changes the interface for running the docker build. Previously a user would provide one variable, now they need to provide 3.

I'm not against it, just want to be sure that the need to specify extra variables wont break anyone.

Reply:
Actually, in the link from your comment below, it looks like docker will automatically populate these variables based on TARGETPLATFORM?

Reply:
that's correct, this is not something we are passing in.